firefox_puppeteer/base.py

```diff
@@ -23,34 +21,3 @@ def marionette(self):
 
     def get_marionette(self):
         return self.marionette
-
-
-class UIBaseLib(BaseLib):
-    """A base class for all UI element wrapper classes inside a chrome window."""
```

Comment:
Yes. BaseLib and UIBaseLib being in the same file was causing one of the circular dependencies.